src/index.js

```diff
@@ -179,6 +179,10 @@ class Bitswap {
     return this.engine.wantlistForPeer(peerId)
   }
 
+  ledgerForPeer (peerId) {
```

Comment:
Added, set return type to {Object} since calling it {Ledger} would be technically incorrect, it's a representation of the ledger object, not the object itself.